project/Dependencies.scala

```diff
@@ -214,8 +214,9 @@ object Dependencies {
         Compile.akkaStream,
         Compile.akkaPersistenceTyped,
         Compile.akkaPersistenceQuery,
-        // Only needed for Replicated Event Sourcing over gRPC
-        Compile.akkaClusterShardingTyped % Optional)
+        // Only needed for Replicated Event Sourcing over gRPC,
+        // and ConsumerFilter with Cluster on consumer side
+        Compile.akkaClusterShardingTyped % "provided")
```

Comment:
I think we discussed this elsewhere, maybe in r2dbc, we should use the same as in Akka since we know that works for both maven and sbt.